schema/attribute/phonenumber-countrycode.json

```diff
@@ -1,7 +1,9 @@
 {
-  "id": "http://platform.selfkey.org/schema/attribute/phonenumber_countrycode.json",
-  "$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
-  "title": "Phone Number",
-  "description": "An individual's phone number",
-  "type": "string"
+	"$id": "http://platform.selfkey.org/schema/attribute/phonenumber_countrycode.json",
```

Comment:
Let's rename this to `phone-number`, make it an object with a `number` attribute and remove reference to country code. We can create a new attribute type extending this one if we want to include country code, internal extension etc.

Reply:
@dfcowell simple phone number can include country code but does not have to? In general I would prefer to not have local phone numbers... it makes the phone attribute useless to most of the world witch is not desirable for web since it's word wide

Reply:
renamed to phone-number